src/app/components/SearchResultsPage/SearchResultsPage.jsx

```diff
@@ -143,7 +135,7 @@ const SearchResultsPage = (props, context) => {
                     total={totalHits}
                     perPage={50}
                     page={parseInt(page, 10)}
-                    to={{ pathname: urlSearchString }}
```

Comment:
`<Link>` takes either a string url or an object in the form of `{ pathname : url }`. So it won't break.